src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const tHead = document.querySelector('thead');
+const tFoot = document.querySelector('tfoot');
+const tBody = document.querySelector('tbody');
+const neededTh = document.querySelectorAll('th');
+const neededTd = document.querySelectorAll('td');
+
+tHead.firstElementChild.insertBefore(neededTh[1].cloneNode(true), neededTh[4]);
+
+tFoot.firstElementChild.insertBefore(neededTh[6].cloneNode(true), neededTh[9]);
+
+let copiedItemCount = 1;
+let insertBeforeItemCount = 4;
+
+for (let i = 0; i < tBody.children.length; i++) {
+  tBody.children[i].insertBefore(neededTd[copiedItemCount]
+    .cloneNode(true), neededTd[insertBeforeItemCount]);
+  copiedItemCount += 5;
+  insertBeforeItemCount += 5;
+}
```

Comment:
or you could iterate through each row and copy it

```suggestion
const tr = document.querySelectorAll('tr');

  for (const row of tr) {
    row.insertBefore(
      row.cells[1].cloneNode(true),
      row.cells[4]
    );
  }
```